patches/chromium/fix_media_key_usage_with_globalshortcuts.patch

```diff
@@ -74,52 +90,20 @@ index 5938f75742b793868638e693a9a8c8dc686dfc46..1263d679a5174beb960265989c370dd4
 -  } else {
 -    // If we can't access system media controls, then directly listen for media
 -    // key keypresses instead.
+-    media_keys_listener_ = ui::MediaKeysListener::Create(
+-        this, ui::MediaKeysListener::Scope::kGlobal);
+-    DCHECK(media_keys_listener_);
+-  }
 +  // This is required for proper functioning of MediaMetadata.
 +  system_media_controls_->AddObserver(this);
 +  system_media_controls_notifier_ =
 +      std::make_unique<SystemMediaControlsNotifier>(
 +          system_media_controls_.get());
 +
 +  // Directly listen for media key keypresses when using GlobalShortcuts.
-+#if defined(OS_MACOS)
```

Comment:
I have a feeling that this define wasn't even working because it is `OS_MAC` :joy: